compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -1066,20 +1066,20 @@ declare_lint! {
     ///
     /// ### Explanation
     ///
-    /// Creating a reference to an insufficiently aligned packed field is
-    /// [undefined behavior] and should be disallowed.
-    ///
-    /// This lint is "allow" by default because there is no stable
-    /// alternative, and it is not yet certain how widespread existing code
-    /// will trigger this lint.
-    ///
-    /// See [issue #27060] for more discussion.
+    /// Creating a reference to an insufficiently aligned packed field is [undefined behavior] and
+    /// should be disallowed. Using an `unsafe` block does not change anything about this. Instead,
+    /// the code should use raw pointers and unaligned accesses, or do a copy of the data in the
```

Comment:
```suggestion
    /// the code should do a copy of the data in the packed field or use raw pointers and unaligned accesses
```
From my experience the most common way to trigger this lint is passing a packed field to a formatting/printing macro that implicitly takes references, like `println!("{:?}", packed.field);`, so it's preferable to put the advice about copying the value first.